UI-extra/index.html

```diff
@@ -153,23 +164,25 @@ <h1 class>LENOVO FLEX, Intel Core i3 14" FHD Multi Point Touch</h1>
                             </div>
                         </div>
                         <hr></hr>
-                        <div class="bundle-item flex-box flex-box—start padding-top-s">
+                        <div class="bundle-item flex-box flex-box__start padding-top-s">
```

Comment:
When a component is used in adjacent with each other, it's preferred to do the spacing the CSS than adding padding/margin to space them out. Ideally we only add padding/margin utility classes to space out *different* components.  So for this guy, we can remove `padding-top-s` and do something like `.bundle-item + .bundle-item` styling